compiler/runtime/OMRRuntimeAssumptions.hpp

```diff
@@ -185,6 +187,59 @@ class PatchNOPedGuardSite : public OMR::LocationRedirectRuntimeAssumption
    uint8_t *_destination;
    }; // TR::PatchNOPedGuardSite
 
+class PatchSites
+   {
+   private:
+   int       _refCount;
+   int       _size; 
+   uint8_t **_patchPoints;
+
+   public:
+   PatchSites(TR_PersistentMemory *pm, int size);
+
+   int getSize() { return _size; }
+   uint8_t *getLocation(size_t index) { return _patchPoints[index * 2]; }
+   uint8_t *getDestination(size_t index) { return _patchPoints[index * 2 + 2]; }
```

Comment:
+2 ? why isn't this +1 given the pairs pattern elsewhere?

Reply:
This was a mistake and has been corrected.